package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "5.6.0",
+  "version": "5.6.1",
```

Comment:
This is typically handled by our promotion scripts - can you please revert the version change?